layouts/partials/blog/archive.html

```diff
@@ -0,0 +1,18 @@
+<section>
+    {{ $prev := 2100}} <!--start at year 2100-->
+    {{range .Pages.ByPublishDate.Reverse}}
+      {{if .Date}}
+      {{$curr := .Date.Format "2006"}}
+        {{if gt $prev ($curr)}}
```

Comment:
I’m not sure I understand this approach. What’s the purpose of using `2100`?

If the goal is to show each previous year, could you not then do `{{ if lt (.Date.Format “2006”) (now.Format "2006") }}`?

Reply:
I used the 2100 year as a hack because I don't know hugo, just used it on the same day I added that archive thingy. - I didn't know about that `now Date` 😄   

I added a change which needs to have the h2 repeated at the top for the current year, I think your suggestion is way better than the first hack I did.